cpp/src/parquet/encoding.cc

```diff
@@ -3055,6 +3149,14 @@ std::unique_ptr<Encoder> MakeEncoder(Type::type type_num, Encoding::type encodin
             "DELTA_BINARY_PACKED encoder only supports INT32 and INT64");
         break;
     }
+  } else if (encoding == Encoding::DELTA_LENGTH_BYTE_ARRAY) {
+    switch (type_num) {
+      case Type::BYTE_ARRAY:
+        return std::unique_ptr<Encoder>(new DeltaLengthByteArrayEncoder(descr, pool));
+      default:
+        throw ParquetException("DELTA_LENGTH_BYTE_ARRAY only supports BYTE_ARRAY");
+        break;
```

Comment:
this line is redundant

Reply:
I'm not sure which one you mean.

Reply:
I mean that a `break` after `throw` is redundant. Will the complier complain if `break` is removed?

Reply:
Good point. Removed.